src/main/java/Tyler.java

```diff
@@ -0,0 +1,189 @@
+
+import java.io.IOException;
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.File;
+import java.io.FileNotFoundException;
+
+import java.nio.file.Paths;
+
+import java.time.format.DateTimeFormatter;
+import java.time.LocalDateTime;
+
+public class Tyler {
+    private static ArrayList<Task> taskList = new ArrayList<>();
+    private static int curr = 0;
+    private static final String FILE_PATH = Paths.get(".",  "data", "Tyler.txt").toString();
+    private static final DateTimeFormatter INPUT_DATE_FORMAT = DateTimeFormatter.ofPattern("yyyy-MM-dd HHmm");
+    private static final DateTimeFormatter OUTPUT_DATE_FORMAT = DateTimeFormatter.ofPattern("dd MMM yyyy h:mm a");
+    public static void main(String[] args) throws IOException {
+        Tyler.loadTask();
+        Tyler.greet();
+        Scanner sc = new Scanner(System.in);
+
+          while (true) {
+              try {
+                  System.out.println("    --------------------------------------------------");
+                  String input = sc.nextLine();
+                  String[] arr = input.split(" ");
+                  if (arr[0].equals("bye")) {
+                      Tyler.saveTask();
+                      System.out.println("    Bye. Hope to see you again");
+                      return ;
+                  } else if (arr[0].equals("list")) {
+                      Tyler.list();
+
+                  } else if (arr[0].equals("mark")) {
+                      if (arr.length < 2) {
+                          throw new MarkNothingException();
+                      }
+                      int taskNumber = Integer.parseInt(arr[1]);
+                      taskList.get(taskNumber - 1).mark();
+                  } else if (arr[0].equals("unmark")) {
+                      if (arr.length < 2) {
+                          throw new MarkNothingException();
+                      }
+                      int taskNumber = Integer.parseInt(arr[1]);
+                      taskList.get(taskNumber - 1).unmark();
+                  } else if (arr[0].equals("todo")) {
+                      if (arr.length < 2) {
+                          throw new EmptyNameException();
+                      }
+                      String[] rest = input.split(" ", 2);
+                      Task newTask = new Todo(rest[1]);
+                      taskList.add(newTask);
+                      System.out.println("    Got it! I've added this task:");
+                      System.out.println("      " + newTask.toString());
+                      curr++;
+                      System.out.println("    Now you have " + curr + " tasks in the list");
+
+                  } else if (arr[0].equals("deadline")) {
+                      if (arr.length < 2) {
+                          throw new EmptyNameException();
+                      }
+                      String[] rest1 = input.split(" ", 2);
+                      String[] rest2 = rest1[1].split(" /by ");
```

Comment:
Could there be a better way to differentiate the variables instead of 1 and 2? The same naming convention can be seen in other parts of the code as well.